module/zfs/zio.c

```diff
@@ -2039,12 +2039,14 @@ zio_taskq_dispatch(zio_t *zio, zio_taskq_type_t q, boolean_t cutinline)
 
 	/*
 	 * If this is a high priority I/O, then use the high priority taskq if
-	 * available.
+	 * available or cut the line otherwise.
 	 */
-	if ((zio->io_priority == ZIO_PRIORITY_NOW ||
```

Comment:
Just confirming that you really meant to remove `ZIO_PRIORITY_NOW`. I think it's fine; the ZIOs that are issued with this priority never had high-pri taskqs anyway. I just didn't see it mentioned so thought to point it out.

Reply:
Yes, it was intentional.  That priority for write was used by vdev cache layer, that was removed not so long ago.  In other use cases this priority is used for all I/Os, and so useless as marker of high priority.  If we'd like to keep it here for some reason -- I don't mind, but I'd need to wrap the TQ_FRONT into another set of #ifdef illumos, that I decided not to.

Reply:
Nah, don't think it's worth it. Thanks!